godot-core/src/builtin/meta/class_name.rs

```diff
@@ -13,12 +13,17 @@ use crate::obj::GodotClass;
 
 /// Utility to construct class names known at compile time.
 /// Cannot be a function since the backing string must be retained.
-#[derive(Eq, PartialEq, Hash, Clone, Debug)]
+#[derive(Default, Eq, PartialEq, Hash, Clone, Debug)]
 pub struct ClassName {
     backing: StringName,
 }
 
 impl ClassName {
+    /// In Godot, an empty `StringName` in a place that expects a class name, means that there is no class.
+    pub fn none() -> Self {
+        Self::default()
+    }
```

Comment:
Sorry, I meant "alternatively" as in "not providing default".

This avoids accidental implicit construction (we also don't need 2 methods doing the same thing here).

```suggestion
    /// In Godot, an empty `StringName` in a place that expects a class name, means that there is no class.
    pub fn none() -> Self {
        Self {
            backing: StringName::default(),
        }
    }
```